### PR TITLE
Windup 3997 mtr 1 2 0 release notes hvider 

### PR DESCRIPTION
**Pull request by anarnold97**
[Windup-3997](https://issues.redhat.com/browse/WINDUP-3997)

[PREVIEW](https://deploy-preview-746--windup-documentation.netlify.app/docs/release-notes-mtr/master/)

I have reverted the erroneous commit in Hagay's PR (see [Hagay's PR)](https://github.com/windup/windup-documentation/pull/736).

I think Phil likes to include earlier releases in his JIRA filter for known issue. It is consistent with filters he has provided previously.

### CODE REVIEW

**Comment by anarnold97** [comment]
@sbeskin-redhat - made your review edits. Please can you give me an LGTM if you are happy
@sshvetahttps - reverted the incorrect commit. Please can you give me an LGTM if you are happy

**Comment by sbeskin-redhat** [comment]
@anarnold97 @HagayVider1 
The Introduction says "These release notes cover all releases of MTR with the most recent release listed first.", which is incorrect. I think it should be:
"These release notes cover all Z-stream releases of MTR 1.2 with the most recent release listed first."

File: docs/topics/mtr-rn-known-issues-1-2-0.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-known-issues-1-2-0_{context}"]
+
+= Known issues
+
+For a complete list of all known issues, see the list of link:https://issues.redhat.com/browse/WINDUP-3470?filter=12419394[MTR 1.2.0 known issues] in Jira.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-known-issues-1-2-0.adoc:10]: @anarnold97 @HagayVider1 
This link is wrong.

File: docs/topics/mtr-rn-resolved-issues-1-2-0.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/mtr_release_notes-1.0/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-resolved-issues-1-2-0_{context}"]
+= Resolved issues
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/WINDUP-3632?filter=12419395[{ProductShortName} 1.2.0 resolved issues] in Jira.
```
Comment (sbeskin-redhat) [inline docs/topics/mtr-rn-resolved-issues-1-2-0.adoc:9]: @anarnold97 @HagayVider1 
This link is wrong.

Comment (anarnold97) [reply docs/topics/mtr-rn-resolved-issues-1-2-0.adoc]: Fixed - https://issues.redhat.com/browse/WINDUP-3632?filter=12419395

Comment (sbeskin-redhat) [reply docs/topics/mtr-rn-resolved-issues-1-2-0.adoc]: @anarnold97 
But this seems to be the same link?

**Pull request merged**